app/views/static_pages/contact.html.erb

```diff
@@ -0,0 +1,16 @@
+<div class="jumbotron mt-4 alert alert-info offset-md-2 col-md-8 offset-md-2">
+  <h1 class="text-center">Inquiry</h1>
+    <p class="text-center">Please feel free to contact us.</p>
+
+  <form>
+    <div class="form-group">
+      <label for="exampleFormControlInput1">Email address</label>
+      <input type="email" class="form-control" id="exampleFormControlInput1" placeholder="name@example.com">
+    </div>
+    <div class="form-group">
+      <label for="exampleFormControlTextarea1">Comment</label>
+      <textarea class="form-control" id="exampleFormControlTextarea1" rows="3"></textarea>
+    </div>
+  </form>
+  <img class="d-block mx-auto" src="home.webp" width="500" height="300">
```

Comment:
please change this code to the following:
`<%= image_tag ("home.webp", width: 500, height: 300, class:"d-block mx-auto") %>`